client/src/modules/list/components/Items/ListItem/ListItem.scss

```diff
@@ -17,14 +19,14 @@
       &::after {
         background-color: $success;
         border-color: $success;
-        background-image: url('../../assets/images/check-solid.svg');
+        background-image: url('../../../../../assets/images/check-solid.svg');
         background-size: 18px;
         background-position: center;
       }
 
       &:hover {
         &::after {
-          background-image: url('../../assets/images/undo-alt-solid.svg');
+          background-image: url('../../../../../assets/images/undo-alt-solid.svg');
```

Comment:
Please inspect is some of the assets can be moved nearby the components that use tham so the path will be up to two levels up.